src/theory/builtin/kinds.toml

```diff
@@ -0,0 +1,445 @@
+# kinds [for builtin theory]                                          -*- sh -*-
+#
+# This "kinds" file is written in a domain-specific language for
+# declaring cvc5 kinds.  Comments are marked with #, as this line is.
+#
+# The first non-blank, non-comment line in this file must be a theory

```

Comment:
The documentation of how kinds files are structured is still the old one, right? Just as a reminder that this needs to be updated to the new format.